### DISCUSSION TITLE
navbar custom  button

### DISCUSSION THREAD

**Discussion by paingchan** [filamentphp/filament#12387] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

<img width="1226" alt="image" src="https://github.com/filamentphp/filament/assets/54994337/535f2a6a-bfe7-480d-9473-42ae4dcc82c0">
 
How can i add custom button in Navigation Bar ?

**Comment by leandrocfe** [comment, marked as answer]
Use render hooks
https://filamentphp.com/docs/3.x/panels/configuration#render-hooks

There is also a plugin 
https://filamentphp.com/plugins/awcodes-quick-create

**Reply by paingchan** [reply]
Thank you it working now.

**Reply by MuhammedKkhaled** [reply]
@paingchan Please How did u  add localization button
